pkgs/applications/networking/browsers/qutebrowser/default.nix

```diff
@@ -55,13 +67,17 @@ in mkDerivationWith python3Packages.buildPythonApplication rec {
     docbook_xml_dtd_45 docbook_xsl libxml2 libxslt
   ];
 
-  propagatedBuildInputs = with python3Packages; [
+  propagatedBuildInputs = with python3Packages; ([
     pyyaml backendPackage jinja2 pygments
     pypeg2 cssutils pyopengl attrs setuptools
     # scripts and userscripts libs
     tldextract beautifulsoup4
     pyreadability pykeepass stem
-  ];
+    # extensive ad blocking
+    adblock
+  ]
+    ++ lib.optional pythonOlderThan "3.9" importlib-resources
```

Comment:
Testing in my excitement revealed nixos-unstable doesn't like this line.

pythonOlderThan seems to exist as pythonOlder, and i had to surround it in parenthesis for it to evaluate.

Does that sound right?

Reply:
Yes, the parenthesis are certainly missing.

Reply:
```suggestion
    ++ lib.optional (pythonOlderThan "3.9") importlib-resources
```